client/src/components/Project.js

```diff
@@ -65,13 +64,13 @@ const Project = ({ data }) => {
             {title}
           </Typography>
           <Typography element="h3" className={classes.funding}>
-            ${currentFunding}
-            <span className={classes.goal}> / {fundingGoal}</span>
+            $12,345
+            <span className={classes.goal}> / {format(fundingGoal)}</span>
           </Typography>
           <Typography element="h4" className={classes.footer}>
-            Equity exchange: {equity}%
+            Equity exchange: 10%
             <span style={{ margin: '0 16px' }}>|</span>
-            {daysLeft} days to go
+            12 days to go

```

Comment:
You'd probably want these to stay as props no?

Reply:
True, we haven't added the time limit to the projects model yet so I hardcoded them for now.